manifests/params.pp

```diff
@@ -73,6 +73,7 @@
   $request_memory                 = hiera('request_memory', true)
 
   $starter_job_environment        = hiera_hash('starter_job_environment', {})
+  $manage_selinux                 = hiera('manage_selinux', true)
```

Comment:
Since this is quite the change, would it be possible to set the default value to `false`?
I assume that otherwise, things will break for anyone who is unaware of this addition.

Reply:
My reasoning for this intrusive change was, since SELinux is on by default on RedHat systems, it's better not to ship a broken default configuration of HTCondor on a default-configured system (which is the case at the moment). 
Of course, I can also flip that to false, and maybe add something to the `README.md` to inform users that on a default RedHat system with SELinux on, `manage_selinux` should be set to true? This would be less intrusive.

Reply:
A related message from upstream:
https://www-auth.cs.wisc.edu/lists/htcondor-users/2017-November/msg00008.shtml
HTCondor will likely (in a future relase) fix a broken SELinux context setting by default (it only ships with correct SELinux contexts on the `pool` directory if that is put at the default location).

Reply:
hmm..that is a good point. Maybe other users can chip in. @fschaer, @kashif74 & @afortiorama what do you think?

Reply:
Maybe to add on this, in case @fschaer , @kashif74 or @afortiorama want to reproduce: The issues caused by a wrong SELinux context on the `pool`-directory only become apparent (at least in my experience up to now) when running interactive jobs in HTCondor, or using `condor_ssh_to_job` to connect to running jobs. 
The issue is that `ssh-keygen` is forbidden by SELinux to write to the `pool`-directory. HTCondor has rules to fix that since the latest release, but they only apply to the default `pool` directoyr location.